internal/exportimport/database/exportimport.go

```diff
@@ -71,7 +74,7 @@ func (db *ExportImportDB) ListConfigs(ctx context.Context) ([]*model.ExportImpor
 				id, index_file, export_root, region, from_timestamp, thru_timestamp
 			FROM
 				exportimport
-			ORDER BY id DESC
+			ORDER BY id ASC
```

Comment:
Hmm - this is a behavior change. Previously they were listed with the newest at the top. Now they'll be listed with the newest at the bottom. That feels wrong.

Reply:
yes - this is for admin console display only.
In the only known production usage of this, there are exactly 3 entries in this table.